vllm/model_executor/layers/quantization/modelopt.py

```diff
@@ -463,6 +478,18 @@ def __init__(self, quant_config: ModelOptNvFp4Config):
         self.quant_config = quant_config
         self.cutlass_nvfp4_supported = cutlass_fp4_supported()
         self.use_marlin = False
+        self.allow_flashinfer_cutlass = False
+
+        if envs.VLLM_USE_FLASHINFER_MOE:
+            if self.cutlass_nvfp4_supported and current_platform.is_cuda() \
+               and current_platform.has_device_capability(10, 0):
```

Comment:
Use `is` since we only support SM100
```suggestion
               and current_platform.is_device_capability(100):
```